### PR TITLE
[25.1] Final changes for 25.1rc1

### PR DESCRIPTION
**Pull request by fanquake**
Final changes to tag a `25.1rc1`.
Bumps version numbers, man pages, adds release notes etc.

### CODE REVIEW

**Comment by fanquake** [comment]
Rebased for the final backports. This is now ready for review.

**Review by dergoegge — APPROVED** [review]
ACK 10f3f813b251d16c86d0b41bbdd7b7ab4a22e00d

File: doc/release-notes.md [hunk 1]
```diff
@@ -40,301 +40,69 @@ unsupported systems.
...
-- Ryan Ofsky
-- S3RK
-- Sebastian Falbesoner
-- Seibart Nedor
-- sinetek
-- Sjors Provoost
-- Skuli Dulfari
-- SomberNight
-- Stacie Waleyko
 - stickies-v
```
Comment (hebasto) [inline doc/release-notes.md:104]: nit: Mention by name, as all others contributors?

**Review by hebasto — APPROVED** [review]
ACK 10f3f813b251d16c86d0b41bbdd7b7ab4a22e00d, I have reviewed the code and it looks OK.

File: doc/release-notes.md (hunk 1 above)
Comment (fanquake) [reply doc/release-notes.md:104]: Copied from the previous release notes.

Comment (fanquake) [reply doc/release-notes.md:104]: There are also other usernames in here.

Comment (hebasto) [reply doc/release-notes.md:104]: > There are also other usernames in here.

image

Comment (fanquake) [reply doc/release-notes.md:104]: > MacroFake

**Review by willcl-ark — APPROVED** [review]
ACK 10f3f813b251d16c86d0b41bbdd7b7ab4a22e00d

Mechanical changes all look correct. Release note reads well.

**Review by stickies-v — APPROVED** [review]
ACK 10f3f813b251d16c86d0b41bbdd7b7ab4a22e00d

I verified the release notes correspond with the the commits since 8105bce5b384c72cf08b25b7c5343622754e7337 , and the manpages and bitcoin.conf correspond with my output.

<details>
<summary>Two minor fixups in `release-notes.md` -> `release-notes-25.0.md`:</summary>

```sh
% diff doc/release-notes.md doc/release-notes/release-notes-25.0.md
46c46
< - Transactions of non-witness size 65 and above are now allowed by mempool
---
> - Transactions of non-witness size 65 bytes and above are now allowed by mempool
63c63
<   parameter](JSON-RPC-interface.md#parameter-passing) called `args` that can
---
>   parameter](https://github.com/bitcoin/bitcoin/blob/master/doc/JSON-RPC-interface.md#parameter-passing) called `args` that can
```
</details>

**Comment by fanquake** [comment]
> Two minor fixups in

Going to leave as-is for now, as the change is move-only. Can fixup pre-release if we want.

**Comment by fanquake** [comment]
> Going to leave as-is for now, as the change is move-only. Can fixup pre-release if we want.

I misinterpreted this as we need to fix somethingup, but it was actually just pointing out the diff. Nothing to followup on.

**Pull request merged**